package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-openshift-templates",
   "description": "OpenShift V3 Templates for FH MBaaS",
-  "version": "0.24.87",
+  "version": "0.24.88",
```

Comment:
@laurafitzgerald @grdryn  happy with either tbh. Not sure this version is really used for anything other than updating the templates in fhcap